tests/functional/test_install_force_reinstall.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+from tests.lib import assert_all_changes, pyversion_tuple
+
+
+def check_installed_version(script, package, expected):
+    # Python 3.3 emits a "Python 3.3 support has been deprecated" warning.
+    expect_stderr = (pyversion_tuple[:2] == (3, 3))
+    result = script.run('pip', 'show', package, expect_stderr=expect_stderr)
+    lines = result.stdout.splitlines()
+    for line in lines:
+        if line.startswith('Version: '):
+            version = line.split()[-1]
+            break
```

Comment:
This should probably fail if it doesn't find version with a nice message instead of a NameError...